scripts/release/upload-artifacts.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -eu
+set -o pipefail
+
+RELEASE=$1
+VERSION=${RELEASE#cabal-install-v}
+URL=https://downloads.haskell.org
+
+cd "gh-release-artifacts/cabal-${VERSION}"
+
+sftp $URL <<EOF
+cd cabal
+
+mkdir cabal-install-$VERSION
+cd cabal-install-$VERSION
+
+put -r .
+
+rm cabal-install-latest
```

Comment:
I think this needs to be done in the parent directory (we're symlinking the directory created by the `mkdir` above, which we're currently in due to the `cd`).